tracee-ebpf/Makefile

```diff
@@ -61,6 +61,10 @@ build: $(OUT_BIN)
 go_env := GOOS=linux GOARCH=$(ARCH:x86_64=amd64) CC=$(CMD_CLANG) CGO_CFLAGS="-I $(abspath $(LIBBPF_HEADERS))" CGO_LDFLAGS="$(abspath $(LIBBPF_OBJ))"
 ifndef DOCKER
 $(OUT_BIN): $(LIBBPF_HEADERS) $(LIBBPF_OBJ) $(OUT_BPF_CORE) $(filter-out *_test.go,$(GO_SRC)) $(BPF_BUNDLE) | $(OUT_DIR)
+	# workaround until Makefile.one becomes standard
```

Comment:
This is needed because embedded.go is at the root directory, embedding dist there, and if we build with regular existing Makefiles, then we have to copy the entire content to it for the embed to work.

I don't want to touch existing Makefile very much, and, with this small workaround we can maintain both working until Makefile.one becomes the standard Makefile.